src/config/Application.js

```diff
@@ -0,0 +1,16 @@
+const AppQuestions = {
```

Comment:
Not sure if this file makes sense in the application, as moving forward we're going to keep things like this inside of the database, configurable by an admin interface. The config folder should be configuration settings that affect the running of the application instance (currently it doesn't follow this rule with how interview questions are stored there, but that will change once the database is ready).